src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -249,9 +249,9 @@ void GcsActorScheduler::LeaseWorkerFromNode(std::shared_ptr<GcsActor> actor,
 
 void GcsActorScheduler::RetryLeasingWorkerFromNode(
     std::shared_ptr<GcsActor> actor, std::shared_ptr<rpc::GcsNodeInfo> node) {
-  execute_after(io_context_,
-                [this, node, actor] { DoRetryLeasingWorkerFromNode(actor, node); },
-                RayConfig::instance().gcs_lease_worker_retry_interval_ms());
+  RAY_UNUSED(execute_after(
```

Comment:
What is this?

Reply:
`execute_after` returns a timer instance, so I add a `RAY_UNUSED` here.

Reply:
I mean is this relevant to this PR?